<?xml version="1.0"?>
<launch>
  <arg name="vector_map_topic" default="/map/vector_map"/>
  <arg name="input/cluster" default="input/clusters"/>
  <arg name="output/cluster" default="output_clusters"/>

  <node pkg="detected_object_validation" exec="cluster_lanelet_filter_node" name="cluster_lanelet_filter" output="screen">
    <remap from="input/vector_map" to="$(var vector_map_topic)"/>
    <remap from="input/cluster" to="$(var input/cluster)"/>
    <remap from="output/cluster" to="$(var output/cluster)"/>
  </node>
</launch>